### PR TITLE
Update amis to run from prod jenkins accounts

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Update amis to run from prod jenkins accounts

### Issues Resolved
https://github.com/opensearch-project/opensearch-ci/issues/204

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by prudhvigodithi — APPROVED** [review]

**Comment by prudhvigodithi** [comment]
We will merge this after the PR https://github.com/opensearch-project/opensearch-ci/pull/186 is merged.
@rishabh6788 @gaiksaya @peterzhuamazon

**Comment by peterzhuamazon** [comment]
> We will merge this after the PR #186 is merged. @rishabh6788 @gaiksaya @peterzhuamazon

Hi @prudhvigodithi feel free to block this PR just in case.

**Pull request merged**
